examples/material_gallery/lib/demo/weathers_demo.dart

```diff
@@ -0,0 +1,495 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:ui' as ui;
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_sprites/flutter_sprites.dart';
+
+ImageMap _images;
```

Comment:
Having them be global is ok, but they should be initialised outside the class then. The problem is having them be initialised in the class (so they get reinitialised each time the class is created), but then having them live outside the class.